src/core/control/settings/Settings.h

```diff
@@ -858,11 +869,17 @@ class Settings {
     ButtonConfig* buttonConfig[BUTTON_COUNT]{};
 
     /**
-     * Which gui elements are hidden if you are in Fullscreen mode,
-     * separated by a colon (,)
+     * View-modes, 0=default, 1=fullscreen, 2=presentation
+     */
+    size_t activeViewMode;
```

Comment:
Type should be `enum ViewMode`

Reply:
Same as first [change-request](https://github.com/xournalpp/xournalpp/pull/4514#discussion_r1057764129)